lib/reporting/reports/suppliers/helpers/columns_helper.rb

```diff
@@ -109,9 +109,10 @@ def total
             proc do |line_item|
               total_price = total_excl_vat_and_fees.call(line_item)
               total_fees = total_fees_excl_vat.call(line_item)
+              total_fees_tax = total_vat_on_fees.call(line_item)
```

Comment:
Sorry for the confusion, the name might be confusing here. All of these totals are summed here so it would result the same as suggested:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12879/files/7858757d55d6c47f850fbfbe3169000f1e4cb3c5..25611fd9a578ae1662b1fe567551c6265f3b7b9c#diff-f70ae0cec8eb7b38c2195738e19c2d71caf1c9fe511d7a46910d84e6add0474bR115